src/main/java/uk/ac/ebi/ampt2d/metadata/SpringDataRestConfig.java

```diff
@@ -80,7 +70,9 @@ public void configureRepositoryRestConfiguration(RepositoryRestConfiguration con
                 config.getCorsRegistry().addMapping("/**").allowedMethods("*").allowedOrigins("*");
                 config.exposeIdsFor(
                         Analysis.class,
-                        ReferenceSequence.class,
+                        Assembly.class,
+                        Transcriptome.class,
+                        Gene.class,
```

Comment:
Maybe it is just me, by `Gene` I am thinking more about something else than the reference gene sequence used in an analysis. But I don't have a better name for this class.